src/System.Management.Automation/CoreCLR/CorePsPlatform.cs

```diff
@@ -602,7 +604,190 @@ internal static int NonWindowsGetProcessParentPid(int pid)
             return IsOSX ? Unix.NativeMethods.GetPPid(pid) : Unix.GetProcFSParentPid(pid);
         }
 
+        #region UNIX Create SSH Process
 
+        //
+        // This code is based on GitHub DotNet CoreFx
+        // It is specific to launching the SSH process for use in
+        // SSH based remoting, and is not intended to be general
+        // process creation code.
+        //
+
+#if UNIX
+        private static readonly UTF8Encoding s_utf8NoBom =
+            new UTF8Encoding(encoderShouldEmitUTF8Identifier: false);
```

Comment:
It seems we can use this everywhere. So maybe put this in "right place" as internal?